### PR TITLE
Modal and News Image Container Improvements and Fixes

### PR DESCRIPTION
**Pull request by LeuAlmeida**
A few days ago my team and I noticed a problem reading the description of the images in the news field and corrected these commits.
We also realized that the modal, depending on the device resolution was being overlaid by the global navigation menu, and also corrected.

We are working on improvements and should send some more contributions here :)

### CODE REVIEW

**Comment by LeuAlmeida** [comment]
@jenkins-plone-org please run jobs

**Comment by LeuAlmeida** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
@LeuAlmeida, I have reverted this, because it broke the Jenkins test server. Please don't merge a PR when Jenkins is failing. See for example this [failure on the 5.1 job for your PR](https://jenkins.plone.org/job/pull-request-5.1/132/console).

The problem is that you removed `CHANGES.rst`. I assume this was not intended.  What were you trying to do? If you want to add a changelog entry, please read [these instructions](https://github.com/plone/plone.releaser/blob/master/ADD-A-NEWS-ITEM.rst). In this case, create a file `news/184.bugfix` with as contents something like this:

```
Modal and news image container improvements and fixes.
[LeuAlmeida]
```

Maybe you can describe the fix better yourself.

I don't know if the fix is good: I just notice an error when running buildout.

File: plonetheme/barceloneta/theme/less/behaviors.plone.less [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 .newsImageContainer {
-	float:right;
-	margin: 0 0 @plone-padding-base-vertical @plone-padding-base-vertical;
-	border-radius: @plone-border-radius-base;
-	box-shadow: 0 1px 2px rgba(0,0,0,.17);
-	overflow: hidden;
+	float: right;
+    margin: 0 0 6px 6px;
+    border-radius: 4px;
+    overflow: hidden;
```
Comment (mauritsvanrees) [inline plonetheme/barceloneta/theme/less/behaviors.plone.less]: Did you intentionally hard hardcode these sizes? Do these sizes differ from the `@plone-*` variables that were there? If not, they should stay the same.

Note: I don't know much about the Plone less/css code, I am just wondering.

**Comment by LeuAlmeida** [comment]
Thanks for the advice @mauritsvanrees and I apologize, this will not be repeated.
I am working with a Plone project and will do further testing. I must submit the changes made to this PR with subsequent changes.

**Pull request merged**